db/migrate/20210426201823_change_product_types.rb

```diff
@@ -1,6 +1,6 @@
 class ChangeProductTypes < ActiveRecord::Migration[6.1]
   def up
-    ProductType.create(title: 'Diapers')
+    ProductType.create(title: 'Diapers').products.create([{title: 'Diapers'}, {title: 'Reusable diapers'}])
```

Comment:
This migration is already in master, so you should not change it. Create a new one instead

Reply:
Thanks, done